utility/importer/brave_importer.cc

```diff
@@ -672,6 +672,19 @@ void BraveImporter::ImportReferral() {
   bridge_->UpdateReferral(referral);
 }
 
+bool CanImportURL(const GURL& url) {
+  if (!url.is_valid())
+    return false;
+
+  const char* const kInvalidSchemes[] = {"about", "chrome-extension"};
```

Comment:
"about" is actually valid

Reply:
@darkdh Right, but `about:` URLs from Muon don't always resolve correctly in brave-core. There are two options here, as described in the original issue: one is to filter everything that might not work, the other is to rewrite the URLs. In the interest of getting this done ASAP I went with the first. I personally prefer the second, but don't have a canonical list of which URLs to rewrite, and how.

Reply:
Maybe a different name would be better; like `kUnresolvableSchemes`?

Reply:
I'm fine with what we have for now because we need new build desperately.
We can do rewrite in a follow-up and see `chrome/browser/browser_about_handler.h`

Reply:
@bsclifton naming doesn't matter here